packages/js/components/src/experimental-select-tree-control/select-tree.tsx

```diff
@@ -44,15 +44,13 @@ export const SelectTree = function SelectTree( {
 	...props
 }: SelectTreeProps ) {
 	const linkedTree = useLinkedTree( items );
+	const menuInstanceId = useInstanceId(
+		SelectTree,
+		'woocommerce-select-tree-control__menu'
+	);
```

Comment:
Created an instance id used mostly for the `querySelector` to target the dropdown menu.